src/Modules/Build.lua

```diff
@@ -226,12 +226,109 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		self.spec:SetWindowTitleWithBuildClass()
 		self.buildFlag = true
 	end)
-	self.controls.secondaryAscendDrop = new("DropDownControl", {"LEFT",self.controls.ascendDrop,"RIGHT"}, 8, 0, 120, 20, nil, function(index, value)
-		self.spec:SelectSecondaryAscendClass(value.ascendClassId)
-		self.spec:AddUndoState()
-		self.spec:SetWindowTitleWithBuildClass()
-		self.buildFlag = true
+	--self.controls.secondaryAscendDrop = new("DropDownControl", {"LEFT",self.controls.ascendDrop,"RIGHT"}, 8, 0, 120, 20, nil, function(index, value)
+	--	self.spec:SelectSecondaryAscendClass(value.ascendClassId)
+	--	self.spec:AddUndoState()
+	--	self.spec:SetWindowTitleWithBuildClass()
+	--	self.buildFlag = true
+	--end)
```

Comment:
I'd say add a comment here and in the other place related to `secondaryAscendancy` explaining why is this commented out. May not be obvious to someone trying to contribute a few leagues in the future.

Reply:
Added